src/core/libmaven/adductdetection.cpp

```diff
@@ -42,9 +42,10 @@ AdductDetection::findAdducts(const vector<PeakGroup>& parentIons,
 
         auto compound = parentGroup.getCompound();
         for (const auto adduct : adductsList) {
-            if (*adduct == *(parentGroup.getAdduct()))
-                continue;
-
+            if(parentGroup.getAdduct()){
```

Comment:
Please make a habit of putting spaces before and after braces (unless its braces enclosing a function's parameters).

Reply:
Okay, I will take care of this